Common/cpp/LayoutAnimations/LayoutAnimationsManager.cpp

```diff
@@ -38,6 +56,22 @@ void LayoutAnimationsManager::clearLayoutAnimationConfig(int tag) {
   enteringAnimations_.erase(tag);
   exitingAnimations_.erase(tag);
   layoutAnimations_.erase(tag);
+
+  sharedTransitionAnimations_.erase(tag);
+  for (auto &[key, group] : sharedTransitionGroups_) {
+    bool isCurrentGroupMember = false;
+    int indexInGroup;
```

Comment:
Would be shorter to use `std::remove` here, I think something like this should be enough to replace the whole body of this loop:
```
std::remove(group.begin(), group.end(), tag)
```

Reply:
From "Notes" in https://en.cppreference.com/w/cpp/algorithm/remove
"These algorithms cannot be used with associative containers such as [std::set](https://en.cppreference.com/w/cpp/container/set) and [std::map](https://en.cppreference.com/w/cpp/container/map)"

Reply:
This has to be followed by `std::erase(it_returned_from_remove, group.end())`, since `std::remove` doesn't actually remove the elements, only moves them to the end of the container.
- this should also be faster than the current approach, since you only do one swap and one `pop_back()`, instead of shifting all the elements of `group` after the removed one by one slot to the left
- this however means that the relative order of elements in `group` isn't preserved, if that's what you wanted. In that case, the code would still be cleaner by replacing the `for`-loop with a call to `std::find`